src/output-filestore.c

```diff
@@ -549,3 +546,10 @@ void OutputFilestoreRegister(void)
     SC_ATOMIC_SET(filestore_open_file_cnt, 0);
 #endif
 }
+
+void OutputFilestoreRegisterGlobalCounters(void)
+{
+#ifdef HAVE_NSS
```

Comment:
file extract does not depend on nss, so why would this counter depend on it?

Reply:
This was previously called behind HAVE_NSS guard, as the filestore output does require NSS as sha256 is used during the filename processing.